### PR TITLE
Add forward/backward support for cuDNN clipped ReLU and ELU

### PR DESCRIPTION
**Pull request by tkerola**
I added support for using clipped ReLU and ELU (introduced in cuDNN v6) activation functions via cuDNN.

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

File: tests/cupy_tests/test_cudnn.py [hunk 1]
```diff
@@ -37,6 +42,27 @@ def test_activation_backward(self):
         cupy.cudnn.activation_backward(self.x, self.y, self.g, self.mode)
 
 
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+    'mode': coef_modes,
+}))
+@unittest.skipUnless(cudnn_enabled, 'cuDNN is not available')
```
Comment (okuta) [inline tests/cupy_tests/test_cudnn.py]: Please fix test for cuDNN v4 environemt.

Comment (tkerola) [reply tests/cupy_tests/test_cudnn.py]: Ok, I fixed it.

**Comment by tkerola** [comment]
@okuta I fixed the PR according to your comment.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
Thanks!

**Pull request merged**